### DISCUSSION TITLE
Integration with Zabbix

### DISCUSSION THREAD

**Discussion by johncarterofmars** [nzymedefense/nzyme#622] [General]
I am looking for a way to have alerts be triggered in Zabbix from Nzyme. I checked the /var/log/nzyme.log but I don't think that will work. Generally, I was needing a way to match strings like UNEXPECTED_SSID in a log file and then that could be triggered in Zabbix. 
Does anyone have any suggestions on how I might go about doing this?
Thank you.

**Comment by lennartkoopmann** [comment]
What sort of transports does Zabbix offer? Does it have some sort of an agent?

**Reply by johncarterofmars** [reply]
Thank you for responding and converting this to a discussion. I am still getting a handle on this on Github. 
Zabbix does have an agent. I can be configured with sudo access to have access to anything on the pi it would need.

**Reply by johncarterofmars** [reply]
Just checking in to see if anyone had any thoughts?

**Reply by lennartkoopmann** [reply]
I'm not familiar enough with Zabbix to help here. Currently, the only ways to link it into nzyme are:

* Reading alerts from the `alerts.log` file
* Receiving emails sent by nzyme
* Writing a custom alarm callback for nzyme

**Reply by johncarterofmars** [reply]
I appreciate it. I will work with the alerts.log file and see what I can come up with. Thank you.
